### DISCUSSION TITLE
Question: New Compliment not Appearing

### DISCUSSION THREAD

**Discussion by scohn** [Monika-After-Story/MonikaModDev#8623] [QA - tech support + general] [answered]
I heard the new compliment is "You always put me in a good mood" but I don't see it even though I have the new update.
Do I need to have higher affection? I'm currently at about 599.45.

**Comment by vgamer1991** [comment]
I have the same issue. My affection is currently at 850.8 so...in the same boat as the person who made this post

**Comment by Booplicate** [comment, marked as answer]
It's unlocked after spending 6 months with Monika.
